src/audio/module_adapter/module_adapter.c

```diff
@@ -198,7 +165,7 @@ static int module_adapter_dp_queue_prepare(struct comp_dev *dev)
 			goto err;
 		dp_queue_append_to_list(dp_queue, &mod->dp_queue_ll_to_dp_list);
 
-		/* it will override source pointers set by module_adapter_sink_src_prepare
+		/* it will override source pointers set before
```

Comment:
in line 180 and 222, mod->num_of_sources and mod->num_of_sinks also re-calculated, is it possible to keep one?

Reply:
It is for dp_queue. I am not sure about this. @marcinszkudlinski  is working on dp queue.

Reply:
Yes, it should be changed - creating of "shadow dpQueues" should also be moved to bind/unbind. 
It is not hurting DP processing at the moment and I don't want to delay this PR.  - so I approved the PR and will change it in separate PR

Target solution is Pipeline 2.0 where sink/src will be a main data exchanging API